### ISSUE TITLE
Failing login after rewrite of login page (2016-10-10)

### ISSUE THREAD

**Issue by steve8x8** [steve8x8/geotoad#343]
```
( o ) Logging in as XXXXXX
/.../geotoad/lib/shadowget.rb:89:in `cacheFile': undefined method `+' for nil:NilClass (NoMethodError)
```

This didn't happen before Oct 10, 1800 UTC.

**Comment by steve8x8** [comment]
Cause: Re-ordered parameters to <form> element (plus another <form> to logout).
Fix: Replace your ```lib/auth.rb``` file with the [new uploaded version](https://github.com/steve8x8/geotoad/files/522530/auth.rb.txt).
A maintenance update (3.26.1) will be released ASAP, after more testing.

**Comment by steve8x8** [comment]
Fixed by release 3.26.1.
**All previous releases do not work any longer. You MUST upgrade.**

**Comment by steve8x8** [comment]
Closing.
